versioned/storage/jdbc/src/main/resources/META-INF/services/org.projectnessie.versioned.storage.testextension.BackendTestFactory

```diff
@@ -1,18 +0,0 @@
-#
```

Comment:
Why is this file deleted?

Reply:
In fact it was moved to `versioned/storage/jdbc-tests/src/main/resources/META-INF/services/org.projectnessie.versioned.storage.testextension.BackendTestFactory` – but since the contents changed significantly (because of package renamings), git considers this as a removal + addition.

Reply:
Ah, thanks for the clarification.